charts/zitadel/test/installation/setup.go

```diff
@@ -0,0 +1,36 @@
+package installation
+
+import (
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/jinzhu/copier"
+)
+
+func (s *ConfigurationTest) SetupTest() {
+	clusterKubectl := new(k8s.KubectlOptions)
+	if err := copier.Copy(clusterKubectl, s.KubeOptions); err != nil {
+		s.T().Fatal(err)
+	}
+	clusterKubectl.Namespace = ""
+	// ignore error
+	_ = k8s.KubectlDeleteFromStringE(s.T(), clusterKubectl, `
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: crdb
+`)
+	// ignore error
+	_ = k8s.KubectlDeleteFromStringE(s.T(), clusterKubectl, `
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: crdb
+`)
+	if _, err := k8s.GetNamespaceE(s.T(), s.KubeOptions, s.KubeOptions.Namespace); err != nil {
```

Comment:
Are there no different error types? If there is another situation than just "notfound"?